From 2e00ab8d07b0a24c86dee0b038c244d62fd5a456 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Sat, 20 Sep 2025 17:20:25 +0100
Subject: [PATCH] Update common submodule

  Richard W.M. Jones (2):
      daemon, generator: Use power of 2 for initial size of Hashtbl.create
      mlcustomize/inject_virtio_win.ml: Use viostor.inf instead of guestor

Fixes: https://issues.redhat.com/browse/RHEL-128776
---
 common | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Submodule common b48c7d00..742c5f8b:
diff --git a/common/mlcustomize/inject_virtio_win.ml b/common/mlcustomize/inject_virtio_win.ml
index 7e46f318..07a91fc7 100644
--- a/common/mlcustomize/inject_virtio_win.ml
+++ b/common/mlcustomize/inject_virtio_win.ml
@@ -334,8 +334,13 @@ and ddb_regedits inspect drv_name drv_pciid =
    * one must add keys into the DriverDatabase.
    *)
 
+  let winarch =
+    match inspect.i_arch with
+    | "i386" -> "x86" | "x86_64" -> "amd64"
+    | _ -> assert false in
+
   let drv_inf = "guestor.inf" in
-  let drv_inf_label = drv_inf ^ "_tmp" in
+  let drv_inf_label = sprintf "%s_%s_0000000000000000" drv_inf winarch in
   let drv_config = "guestor_conf" in
 
   [
